AutomatedTest/src/main/resources/application.yml

```diff
@@ -1,11 +1,11 @@
 security:
   basic-auth:
-    password: ${BASIC_AUTH_PASS}
-    username: ${BASIC_AUTH_USER}
+    password: ${BIOMETRIC_AUTH_PASS}
+    username: ${BIOMETRIC_AUTH_USER}
 test:
   host: ${ORDS_HOST}
-  password: ${BASIC_AUTH_PASS}
-  username: ${BASIC_AUTH_USER}
+  password: ${BIOMETRIC_AUTH_PASS}
+  username: ${BIOMETRIC_AUTH_USER}

```

Comment:
automatic test will use BIOMETRIC_AUTH_PASS instead of BASIC_AUTH_USER/PASS to login.